snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: linkerd
+base: core18
+version: '2.7.0'
+summary: Command-line utility to interact with linkerd
+description: |
+    Linkerd is a service mesh, designed to give platform-wide observability,
+    reliability, and security without requiring configuration or code changes.
+
+grade: stable
+confinement: classic
+
+parts:
+    linkerd:
+        plugin: dump
+        source: .
+        override-pull: |
+            curl -sSLO https://github.com/linkerd/linkerd2/releases/download/stable-2.7.0/linkerd2-cli-stable-2.7.0-linux
+            mv linkerd2-cli-stable-2.7.0-linux linkerd # rename binary to linkerd
```

Comment:
The version number is hardcoded for now. I can inject it using environment variables if required.

Reply:
We can add a step in our CI job to `sed` the release number.

Reply:
Yeah we could have. We don't need it now though :)